trustedservices/local.go

```diff
@@ -0,0 +1,238 @@
+// Copyright 2021 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package trustedservices
+
+import (
+	"errors"
+	"fmt"
+	"reflect"
+	"strings"
+
+	"github.com/veraison/common"
+	"github.com/veraison/veraison/kvstore"
+	"google.golang.org/protobuf/types/known/structpb"
+)
+
+func NewLocalClientParamStore() (*common.ParamStore, error) {
```

Comment:
Could you please add few comments here, what is the purpose of this, especially if this is a vts client then why it is termed as local ?